tests/conftest.py

```diff
@@ -122,4 +138,22 @@ def rewrite_docker_compose(content: bytes) -> bytes:
         "command"
     ] = "gunicorn -b 0.0.0.0 -k uvicorn.workers.UvicornWorker config.asgi:application"
 
+    for service in ["celery_worker", "celery_worker_evaluation"]:
+        # Strip watchfiles command from celery
+        # as this is not included in the base container
+        command = spec["services"][service]["command"]
+        command = command.replace('watchfiles "', "")
+        command = command.replace('" /app', "")
+        spec["services"][service]["command"] = command
+
     return yaml.safe_dump(spec).encode("utf-8")
+
+
+def rewrite_makefile(content: bytes) -> bytes:
+    # Using `docker compose` with version 2.4.1+azure-1 does not seem to work
+    # It works locally with version `2.5.1`, so for now go back to docker-compose
+    # If this is fixed docker-compose-wait can be removed and the `--wait`
+    # option added to the "up" action above
+    makefile = content.decode("utf-8")
+    makefile = makefile.replace("docker compose", "docker-compose")
```

Comment:
This will work for now, but care should be taken for in some configurations/OS/frameworks `docker-compose` is just an alias for `docker compose`.

Reply:
Yep, we should hear about it if it breaks.